serverless-logic/serverless-logic-supporting-services/serverless-logic-data-index.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-logic-data-index"]
+= Data index
+include::_attributes/common-attributes.adoc[]
+:context: serverless-logic-data-index
+
+toc::[]
+
+The Data index service is a dedicated supporting service that stores the data related to the workflow instances and their associated jobs. This service provides a GraphQL endpoint allowing users to query and modify that data.
```

Comment:
And similar question here about whether "service" is the Kube API object (see comment on "Job service").